assets/helpers/contributions.js

```diff
@@ -97,3 +100,25 @@ export function billingPeriodFromContrib(contrib: Contrib): BillingPeriod {
     default: return 'Monthly';
   }
 }
+
+export function errorMessage(
```

Comment:
This logic would be better located in a message helper and not scatter around the code. Since there are more messages around the code, we can do this in an upcoming PR.